ts/features/mvl/saga/networking/handleGetMvlDetails.ts

```diff
@@ -40,7 +41,8 @@ const convertMvlAttachment = (
     contentType: attachment.content_type.toLowerCase(),
     resourceUrl: {
       href: generateAttachmentUrl(messageId, attachment.id as MvlAttachmentId)
-    }
+    },
+    size: (attachment as unknown as { size: Byte }).size
```

Comment:
In my understanding this is a placeholder since `Attachment` type doesn't have the `size` field atm and will result always in `undefined`. Could be worth to add a `TODO` just to trace this and explain the reason, what you think?

Reply:
Please note that a TODO is [already in place](https://github.com/pagopa/io-app/pull/3687/files/7ffa1a9e7c24a90fb252688378dec2c3a2fc25cb#diff-4bb6cd40a79a3db147420135270df639339d8f29ad55c71971bf9c5f96de5562R37) for the entire object returned.
The developer will know that the whole `convertMvlAttachment` is a brittle layer of glue between the UI and an API that is [still in the making](https://pagopa.atlassian.net/browse/MVL-63).
I'd refrain from adding another, more specific, TODO if possible.